app.logorrr.LogoRRR.yml

```
@@ -0,0 +1,40 @@
+app-id: app.logorrr.LogoRRR
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: LogoRRR
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host:ro
+  - --persist=~/.logorrr/
```

```suggestion
  - --persist=.logorrr
```

this is the correct syntax for persist